### PR TITLE
Tutorial for visualisation

### PR DESCRIPTION
**Pull request by totetmatt**
Just explaining process to visualise the db. I hope it will give more interest to the project if you can have some pre-visualisation of the data :)

### CODE REVIEW

File: HOWTO_VIZ.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Download the latest version in https://gephi.org/ .
+
+# Get Latest version of database
+
+The history and latest version are available here : https://greetsgraf.jetlag.group/_data/
+
+# Import the GreetGraph to Gephi
+
+- Open Gephi and create a **New Project**
+- Go to **File** >> \*\*
```
Comment (w23) [inline HOWTO_VIZ.md:16]:
```suggestion
- Go to **File** >> Import Database >> Edge List...
```

File: HOWTO_VIZ.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+It should result to this
+
+!['./imgs/HOWTO_04.png'](./imgs/HOWTO_04.png)
+
+# Play with the graph
+
+!['./imgs/HOWTO_05.png'](./imgs/HOWTO_05.png)
+
+# Exemple with sime time spend on it
```
Comment (w23) [inline HOWTO_VIZ.md:55]:
```suggestion
# Example with some time spent on it
```

**Review by w23 — CHANGES_REQUESTED** [review]
Thanks! That's a cool tutorial.

I couldn't replicate it locally, though. For some reason the graph window shows "Initializing..." and no graph ever appears. That is very likely an issue with my local java-wayland-linux interop. Will try on Windows if I get some time.

File: HOWTO_VIZ.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# How to visualise locally the Greet Graf
```
Comment (w23) [inline HOWTO_VIZ.md:1]:
```suggestion
# How to visualize the Greets Graf locally 
```

File: HOWTO_VIZ.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+This is a simple tuto that try to explain how to use Gephi to visualise the Greet Graf.
```
Comment (w23) [inline HOWTO_VIZ.md:3]:
```suggestion
This is a simple tutorial that tries to explain how to use Gephi to visualize the Greets Graf.
```

**Comment by totetmatt** [comment]
> 
> 
> Just explaining process to visualise the db. I hope it will give more interest to the project if you can have some pre-visualisation of the data :)

Yeah the tool is great, but kind of old and know for some bug ...... sorry about that :/

**Comment by w23** [comment]
It seems to explicitly require OpenJDK 8. After downgrading from 11 to 8 it ran. Although I can't figure out how to make it label the groups/nodes.

**Comment by totetmatt** [comment]
> 
> 
> It seems to explicitly require OpenJDK 8. After downgrading from 11 to 8 it ran. Although I can't figure out how to make it label the groups/nodes.

image

**Pull request merged**